### PR TITLE
Remove selenium ca9 oral arguments

### PR DESCRIPTION
**Pull request by flooie**

### CODE REVIEW

File: juriscraper/oral_args/united_states/federal_appellate/ca9.py [hunk 1]
```diff
@@ -3,45 +3,28 @@
...
-            if " " in url:
-                logger.warning(f"URL includes spaces: {url}")
-                continue
-
+        for row in rows[1:-2]:
+            parts = row.xpath(".//td[6]/a/@href")[0].split("/")
+            year = parts[-3][1:5]
+            month = parts[-3][5:7]
+            day = parts[-3][7:]
+            docket_number = parts[-2]
```
Comment (mlissner) [inline juriscraper/oral_args/united_states/federal_appellate/ca9.py:24]: Yowza. Can you explain what all this indexing is doing for the poor sap that comes next?

**Review by mlissner — CHANGES_REQUESTED** [review]
An emerging trend: Need more comments in scrapers.

File: juriscraper/oral_args/united_states/federal_appellate/ca9.py (hunk 1 above)
Comment (flooie) [reply juriscraper/oral_args/united_states/federal_appellate/ca9.py:24]: yessir.

Comment (flooie) [reply juriscraper/oral_args/united_states/federal_appellate/ca9.py]: back to you

Comment (mlissner) [reply juriscraper/oral_args/united_states/federal_appellate/ca9.py]: It's always helpful to have an example as well as an explanation, but this helps. Merging.

**Pull request merged**